pureport/client/model_cloud_region.go

```diff
@@ -10,7 +10,7 @@
 package client
 
 type CloudRegion struct {
-	Name               string `json:"Name"`
+	DisplayName        string `json:"displayName"`
```

Comment:
This is where the API mapping occurs between the REST API and the generated `golfing` code.
We could override this here if needed, but going to keep it the same for now.